tests/config.py

```diff
@@ -35,6 +35,7 @@
 
 fun_names_dict = {
     'get_dictionary_value': 'get_dictionary_value',
+    'modify_list': 'modify_list'

```

Comment:
can you add a comma to the end here like "'modify_list': 'modify_list'," as the other PRs will not clash with yours in that case and there won't be any merge conflicts